tests/lib/dirs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Default applies for: Ubuntu, Debian
+SNAPMOUNTDIR=/snap
+LIBEXECDIR=/usr/lib
+CORELIBEXECDIR=/usr/lib
```

Comment:
I would like it to have a constant in the code replace with a proper variable to express what it really is. If we would be in Go/C this would be marked as const so that it can never be changed.